File: planning/enablement-pack/fixtures/sample_test.go
package main

import (
	"testing"
)

// Sample Go Test Fixture

func Add(a, b int) int {
	return a + b
}

func TestAdd(t *testing.T) {
	result := Add(1, 2)
	if result != 3 {
		t.Errorf("Expected 3, got %d", result)
	}
}

File: planning/enablement-pack/fixtures/sample.test.ts
import { describe, it, expect } from 'vitest'; // or jest

// Sample Typescript Test Fixture
describe('Smoke Test (TS)', () => {
  it('should add numbers correctly', () => {
    const sum = (a: number, b: number) => a + b;
    expect(sum(1, 2)).toBe(3);
  });

  it('should handle async operations', async () => {
    const asyncSum = async (a: number, b: number) => Promise.resolve(a + b);
    expect(await asyncSum(2, 3)).toBe(5);
  });
});
